### PR TITLE
Tutorial: Hosting Static Sites with Wordpress

### PR DESCRIPTION
**Pull request by signalnerve**
This PR adds a tutorial for using Workers Sites and a WP export plugin to deploy a static version of your Wordpress site to Workers. 🎊

### CODE REVIEW

**Comment by signalnerve** [comment]
thanks for the feedback @victoriabernard92! here's what i've updated:

- [x] moved tutorial to `/tutorials/hosting-static-wordpress-sites`
- [x] added `cp` example command
- [x] suggested auth'd preview instead of publish steps, pointed to quick start
- [x] Update /workers/tutorials/ with accurate tutorial links every time you add a tutorial
- [x] Move all media files to appropriate directory

> You have to have a business plan to install plugins, which you sort of mentioned in prereqs but I was confused

not sure about this part. are you talking about flywheel specifically?

**Comment by victoriabernard92** [comment]
@signalnerve
So I was trying to say the following user stories we should consider:
1.  As a user  without any wordpress site I want to use Cloudflare Wordpress from scratch. 
2. As a user using wordpress on a paid hosting platform that doesn't allow plugins for my plan, how can I use Workers Sites

For 1. it is easy to just recommend flywheel or another local Wordpress instance. 2. is a little tricker but maybe also solved through flywheel

File: content/tutorials/hosting-static-wordpress-sites/_index.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+With an export ready of our site, it's time to get to work deploying it. To do this, we'll use [Wrangler](https://github.com/cloudflare/wrangler), the command-line tool for Cloudflare Workers. If you haven't yet installed and configured Wrangler, check out our [Quick Start guide](/quickstart).
+
+Once Wrangler is installed and configured, we can create a new project for deploying our static Wordpress site to Workers. To generate a new project run:
+
+```sh
+wrangler generate --site wp-static
+```
+
+the [`--site`](/workers/tooling/wrangler/sites/#commands) flag indicates that we want to deploy a static site, namely, our static Wordpress site.
```
Comment (victoriabernard92) [inline content/tutorials/hosting-static-wordpress-sites/_index.md]:
```suggestion
the [`--site`](/tooling/wrangler/sites/#commands) flag indicates that we want to deploy a static site, namely, our static Wordpress site.
```
My B

File: content/tutorials/hosting-static-wordpress-sites/_index.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+1. A `public` directory, representing the site you want to deploy to Workers. This will be the location of your Wordpress site.
+2. A `workers-site` directory, which contains the Workers script that will serve your website content. In this tutorial, we won't change anything in this folder.
+3. A `wrangler.toml` file, which contains the configuration details for our `wp-static` file. We'll populate this file with some information before the project is deployed.
+
+When deploying a website using Workers Sites, your static code – your HTML, CSS, and JavaScript – will be uploaded to Workers KV. The location of these files is by default the `public` folder mentioned above.
+
+Open the ZIP file downloaded from your Wordpress static export, and extract the contents of the ZIP into the `public` folder:
+
+```sh
+cp -R ~/Downloads/wp-static-html-output-123/ ./public
```
Comment (victoriabernard92) [inline content/tutorials/hosting-static-wordpress-sites/_index.md]:
```suggestion
cp -R ~/Downloads/wp-static-html-output-<some-tag>/ ./public
```
another My B

**Comment by signalnerve** [comment]
Got the ok from @ashleygwilliams to publish this today. Merging and will publish some promo around it on Twitter in a few hours! 🎊

**Pull request merged**